js/values.js

```diff
@@ -267,6 +312,20 @@ function highlighting(user, baseUrl) {
     $(window).scroll(function(e) {
       var top = annote_position.top - ($(window).scrollTop() - annote_position.anchor_top);
       var left = annote_position.left - ($(window).scrollLeft() - annote_position.anchor_left);
+      
+      //this gets how far down the page you've gone - 
+      //TODO: need to save this number, maybe create a global variable and save it in that, then include it in a post? 
+      // console.log($(window).scrollTop());
+      // console.log($(window).height());
+      // console.log($(document).height());
+      
+      var scrolledDistance = $(window).scrollTop();
+      var windowHeight = $(window).height();
+      var totalHeight = $(document).height();
+
+      var totalScrolledPercent = (scrolledDistance / (totalHeight - windowHeight) ) *100;
+
+      console.log("% scrolled:" + totalScrolledPercent)
```

Comment:
let's not have console logs in production unless you really need it

Reply:
agreed, will take these out during cleanup of comments. Didn't realize my latest push would update the pull request